docs/reference/sql/functions/geo.asciidoc

```diff
@@ -134,6 +134,31 @@ The return type is double.
 include-tagged::{sql-specs}/docs/geo.csv-spec[y]
 --------------------------------------------------
 
+[[sql-functions-geo-st-z]]
+===== `ST_Z`
+
+.Synopsis:
+[source, sql]
+--------------------------------------------------
+ST_Z(geometry<1>)
+--------------------------------------------------
+
+*Input*:
+
+<1> geometry
+
+*Output*: double
+
+.Description:
+
+Returns the the altitude of the first point in the geometry.
+The return type is double.
```

Comment:
I would have added, instead of this short statement that seems to have the same purpose as the `*Output*: double` from above: `Returns the altitude of the first point in the geometry as a double type number`.